numpy/__init__.cython-30.pxd

```diff
@@ -289,14 +289,32 @@ cdef extern from "numpy/arrayobject.h":
         # directly accessing this field.
         cdef char byteorder
         cdef int type_num
-        cdef int itemsize "elsize"
-        cdef int alignment
-        cdef object fields
-        cdef tuple names
+
+        @property
+        cdef inline npy_intp itemsize(self) nogil:
+            # Note: if you need to set the itemsize use PyDataType_SET_ELSIZE
```

Comment:
I don't think we need the comment, no one should be messing with setting itemsize from Cython until we expose a proper API for constructing user-defined dtypes from python.
```suggestion
```

Reply:
There is one use-case where it might be used:  We don't have a proper way to define new strings without it.

But happy to remove it, and see if someone complains (chances are nobody will).

Reply:
You can make the call, whatever you think.

Reply:
Let's try it, it would be nice to not expose every little odd thing to Cython and easy to add.